src/core_scan/Core_scan.ml

```diff
@@ -311,6 +337,11 @@ let print_match ?str (config : Core_scan_config.t) match_ ii_of_any =
      in
      Out.put (spf "%s:%d: %s" file line (String.concat ":" strings_metavars));
      ());
+  Option.iter
+    (fun (toks, version) ->
+      Out.put ("with dependency match at version " ^ version);
+      Core_text_output.print_match ~format:config.match_format toks)
+    dep_toks_and_version;
```

Comment:
I usually do dep_toks_and_version |> Option.iter (...)
so it's easier to see what the code operates on (it's the only thing OO got right).